containers/layouts/LayoutsSection.js

```diff
@@ -50,6 +56,11 @@ const LayoutsSection = () => {
         call();
     };
 
+    const handleToggleStickyLabels = async (value) => {
+        await requestStickyLabels(value);
+        call();
```

Comment:
A notification should be added after this async process.

Reply:
all in `LayoutsSection` now!